.github/workflows/lint.yml

```diff
@@ -21,10 +21,17 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-hack@0.6.21
```

Comment:
Do we need this specified version? What about making the action always install latest `cargo -hack`?

Reply:
Changes in the interface or behavior could break the build or stop checking a particular case; if this happens, I would prefer it not happen without warning.